tests/integration/test_get_item.py

```diff
@@ -24,6 +25,23 @@ def test_basic():
     assert x[2] == 3
 
 
+ARRAYS_4_3_2_1_0 = [
+    4 - num.arange(5),
+    4 - np.arange(5),
+    [4, 3, 2, 1, 0],
+]
+
+
+@pytest.mark.parametrize("arr", ARRAYS_4_3_2_1_0)
+def test_scalar_ndarray_as_index(arr):
+    offsets = num.arange(5)  # [0, 1, 2, 3, 4]
+    offset = offsets[3]  # 3
+    # assert arr[offset] == 1  # TODO: doesn't work when arr is a num.ndarray
```

Comment:
This fails with:

```
Assertion failed: (false), function double_dispatch, file /Users/mpapadakis/legate.core/install/include/core/utilities/dispatch.h, line 239.
```

I think this is because `zip_indices` doesn't handle 0d arrays. @ipdemes we may want to cast 0d arrays to 1d in `zip_indices`, as is done in other places.

Reply:
the fix for this seems to involve more than just casting of array to 1d in ZIP task. I would probably need to do something similar for the Copy task as well since I will have to call copy() on the output of ZIP later during `get_item` operation. I will work on the fix tomorrow.